src/main/java/net/imagej/ops/threshold/ThresholdNamespace.java

```diff
@@ -122,6 +126,24 @@
 		return result;
 	}
 
+	// -- binary mask --
+
+	@OpMethod(op = net.imagej.ops.threshold.binaryMask.DefaultBinaryMask.class)
```

Comment:
there already exist threshold ops in ops. see https://github.com/imagej/imagej-ops/blob/master/src/main/java/net/imagej/ops/threshold/ThresholdNamespace.java. However, we could / should modify these ops to also get a `Condition` (`lt`, `gt`, ... etc) as input parameter. Then we could cover all the cases you implemented here. @stelfrich are you still refactoring threshold ops?